File: iosApp/iosApp/Theme/ProBadge.swift
import SwiftUI

struct ProBadge: View {
    var body: some View {
        Text("PRO")
            .font(.strakkOverline)
            .foregroundStyle(Color.strakkPrimary)
            .padding(.horizontal, 6)
            .padding(.vertical, 2)
            .background(Color.strakkPrimary.opacity(0.15), in: RoundedRectangle(cornerRadius: 4))
    }
}

File: androidApp/src/androidMain/kotlin/com/strakk/android/ui/components/ProBadge.kt
package com.strakk.android.ui.components

import androidx.compose.foundation.background
import androidx.compose.foundation.layout.padding
import androidx.compose.foundation.shape.RoundedCornerShape
import androidx.compose.material3.MaterialTheme
import androidx.compose.material3.Text
import androidx.compose.runtime.Composable
import androidx.compose.ui.Modifier
import androidx.compose.ui.draw.clip
import androidx.compose.ui.text.font.FontWeight
import androidx.compose.ui.unit.dp
import androidx.compose.ui.unit.sp

@Composable
fun ProBadge(modifier: Modifier = Modifier) {
    Text(
        text = "PRO",
        color = MaterialTheme.colorScheme.primary,
        fontSize = 11.sp,
        fontWeight = FontWeight.SemiBold,
        modifier = modifier
            .clip(RoundedCornerShape(4.dp))
            .background(MaterialTheme.colorScheme.primary.copy(alpha = 0.15f))
            .padding(horizontal = 6.dp, vertical = 2.dp),
    )
}
